src/home_assistant.rs

```diff
@@ -90,6 +105,7 @@ impl Config {
             self.switch("ac_charge", "AC Charge")?,
             self.switch("charge_priority", "Charge Priority")?,
             self.switch("forced_discharge", "Forced Discharge")?,
+            self.number_percent("discharge_cutoff_soc_limit_pct", "Discharge Cutoff %", "105")?,
```

Comment:
Really not a fan of how that third argument (holding register number for the state topic)

Reply:
Yeah agreed its a little magic. having said that I did similar by hardcoding `bits/21` in the switches, its ok for now as most of them live there anyway.

I'd probably like to see it passed as `105`, ie an `i16`, rather than a `&str` though. I think `format` will just deal with that automagically.

Maybe in future a refactor can move the list of numbers out to a config hash of names/labels/registers/anything else required.

Reply:
One potential easy improvement is to use the `enum` defined at https://github.com/celsworth/lxp-bridge/blob/master/src/lxp/packet.rs#L538

I think this would then just be a case of replacing 105 (once its an int) with `lxp::packet::Register::DischgCutOffSocEod`, which is a little less magic.